cpp/src/bitmask/null_mask.cu

```diff
@@ -165,7 +165,7 @@ rmm::device_buffer create_null_mask(size_type size,
 // or null, otherwise;
 void set_null_mask(bitmask_type* bitmask, size_type begin_bit, size_type end_bit, bool valid)
 {
-  return detail::set_null_mask(bitmask, begin_bit, end_bit, valid);
+  return detail::set_null_mask(bitmask, begin_bit, end_bit, valid, cudf::get_default_stream());
```

Comment:
Changes like this one should be kept and merged in the short term for consistency with the rest of the library, regardless of what we choose to do in the longer term. Public APIs should not rely on the detail API providing a default stream.

I suspect the only reason this wasn't providing a stream is because it doesn't take a memory resource, so that parameter could be left at default instead of passing in the public API's memory resource.